src/components/PatentUpload/PatentUpload.js

```diff
@@ -1,4 +1,110 @@
-const PatentUpload = () => (
-    <div>Patent Upload Page</div>
-);
+import { Component } from 'react';
+import { Storage } from 'aws-amplify';
+import awsExports from '../.././aws-exports';
+import './PatentUpload.css';
+
+class PatentUpload extends Component {
+  // Initialize state
+  state = {
+    imageName: "",
+    imageFile: "",
+    response: "",
+  };
+
+  uploadPatent = () => {
+    /* First part of function checks user input to ensure it is not blank
+    or include extraneous special characters*/
+    var input = document.getElementById('file');
```

Comment:
nit: new JS variable declaration. if the variable will change use `let` instead of `var`. if the variable will not just use `const`